src/stories/containers/Finances/utils/utils.ts

```diff
@@ -716,3 +717,51 @@ export const sortDataByElementCount = (data: MockData) => {
 
   return sortedData;
 };
+
+export const showOthersFooterRow = (data: MockData) => {
+  let totalRows = 0;
+  Object.keys(data).forEach((key) => {
+    totalRows += data[key].length;
+  });
+  console.log({ totalRows });
```

Comment:
did you forgot to delete this ` console.log` ?

Reply:
yes, done